e2e/cypress/integration/system_console/plugin_upload_spec.js

```diff
@@ -108,6 +110,64 @@ describe('Draw Plugin - Upload', () => {
         cy.findByText(/Installed Plugins/).scrollIntoView().should('be.visible');
         cy.findByTestId('com.mattermost.draw-plugin').should('not.exist');
     });
+
+    it('M27802-MT40-Plugin remains enabled when upgraded', () => {
+    // * upload Demo plugin from the browser
```

Comment:
The test as written here will require the demo plugin to be copied into the 'fixture' folder inside cypress, instead of it getting downloaded from the browser. That will actually be fine as well; however, to note that please add a comment above this test to download both the plugin versions from https://github.com/mattermost/mattermost-plugin-demo/releases (or the direct link to the release bundles within that link). I will also update the test case to reflect that. Thank you!

Reply:
I added the notes to the top of the test spec.  If you prefer it directly above the `it()` block, I can move it.  If you need any additional changes, please let me know!

Reply:
Thanks @jfrerich! I'm good with the notes location.
Hello @prapti, there are 2 steps in TM4J but maybe it's good to combine into one. I'll defer to you since you'll be updating the ticket. We just need to make sure that the number of steps should match between Cypress and TM4J.